src/swagger/accomodation.doc.js

```diff
@@ -0,0 +1,192 @@
+// eslint-disable-next-line import/prefer-default-export
+export const accomodations = {
+  '/api/accommodations/': {
+    post: {
+      tags: ['Accommodations CRUD Operations'],
+      summary: 'Create an accomodation',
+      description: 'Create an accomodation',
+      parameters: [],
+      requestBody: {
+        content: {
+          'multipart/form-data': {
+            schema: {
+              type: 'object',
+              properties: {
+                accommodationName: {
+                  type: 'string',
+                },
+                description: {
+                  type: 'string',
+                },
+                images: {
+                  type: 'file',
+                },
+                amenities: {
+                  type: 'array',
+                },
+                geoCoordinates: {
+                  type: 'array',
+                },
+                streetAddress: {
+                  type: 'string',
+                },
+                locationId: {
+                  type: 'integer',
+                },
+              },
+            },
+          },
+        },
+      },
+      responses: {
+        201: {
+          description: 'Accomodation created successfuly!',
+        },
+        500: {
+          description: 'Server error',
+        },
+      },
+    },
+    get: {
+      tags: ['Accommodations CRUD Operations'],
+      security: [],
+      summary: 'Get all accommodations',
+      description: 'Get all accommodations',
+      responses: {
+        200: {
+          description: 'All accommodations retrieved successfuly!',
+        },
+      },
+    },
+  },
+  '/api/accommodations/in/{locationId}': {
+    get: {
+      tags: ['Accommodations CRUD Operations'],
+      security: [],
+      summary: 'This will retrieve accomodations in given location',
+      description: ' ',
+      produces: ['application/json'],
```

Comment:
This is an OpenApi 2 option. For OpenApi 3 we use the `responses` option. Find other instances where you used this and remove them